app/models/user.rb

```diff
@@ -63,7 +63,7 @@ def self.creation_with_auth(user_params)
   end
 
   def allowed_login?(app_name)
-    if app_name == DONOR_APP
+    if SKIP_AUTH_APP_NAMES.include?(app_name)
```

Comment:
Technically, we're not skipping authentication but we are short-circuiting authorization (both AUTH prefixes so hard to distinguish between them). To make the code a bit clearer, could you remove `SKIP_AUTH_APP_NAMES` and just change to ```[DONOR_APP, BROWSE_APP].include?(app_name)``` - just so things are ultra-clear. The approach you've taken is otherwise fine.

This is a really minor point but I think it will help us going forward to see exactly what this method is doing each time we look at it rather than having to look up a constant definition in another file. Does that make sense?

Reply:
Sure, will add that array .